packages/browser/rollup.config.npm.mjs

```diff
@@ -33,7 +33,7 @@ const plugins = [
     preventAssignment: true,
     values: {
       'process.env.NODE_ENV': JSON.stringify('production'),
-      __BUGSNAG_NOTIFIER_VERSION__: JSON.stringify(packageJson.version),
+      __BUGSNAG_NOTIFIER_VERSION__: packageJson.version,
```

Comment:
I've reverted how the version handling is done across all packages because I'd misunderstood how it was working. As can be seen below the version is injected inside a string rather than as a variable. This is how it was done before. The problem with it being a variable is that outside of rollup you can't import or require the file without somehow defining the global variable first. This is done in some of the integration tests for example.